cfe_internal/enterprise/federation/federation.cf

```diff
@@ -645,13 +672,14 @@ bundle agent ensure_feeders
 
   commands:
     am_superhub::
-      "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh"
+      "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh" -> { "CFE-951" }
         arglist => {
                      "cfdb",
                      `"select ensure_feeders($(feeders_arg));"`
                    },
         classes => psql_wrapper_exit_codes,
-        if => isgreaterthan(length(feeders), 0);
+        if => isgreaterthan(length(feeders), 0),
+        contain => default:in_dir( "/tmp" );
```

Comment:
no need for `/tmp`?

Reply:
Similar to https://github.com/cfengine/masterfiles/pull/2265#discussion_r827273694

inside psql_wrapper.sh.mutache I see `su - cfpostgres`.

```
OUT=$(su - cfpostgres --command "{{{vars.sys.bindir}}}/psql --quiet --tuples-only --no-align --no-psqlrc \"$1\" --command=\"$2\"" 2> $TMP)
```

Reply:
ditto, makes sense to me to put the `cd /tmp` in the psql_wrapper script instead of in policy.